redirects.yaml

```diff
@@ -3,6 +3,7 @@
 # replacing the prefix with the second path.
 
 - /1.0/ /
+- /2.0/ /
```

Comment:
Well, it's not, really. But when poking around I realized that if I edited my URL to go to www.polymer-project.org/2.0/ manually, I just ended up on a 404 page, and I thought this might be better.